pipelines/batch/setup_prod_job.py

```diff
@@ -97,44 +97,50 @@ def main(
         mount_pairs=[
             {
                 "source": "nssp-etl",
-                "target": "/pyrenew-hew/nssp_demo/nssp-etl",
+                "target": "/pyrenew-hew/nssp-etl",
             },
             {
                 "source": "nssp-archival-vintages",
-                "target": "/pyrenew-hew/nssp_demo/nssp-archival-vintages",
+                "target": "/pyrenew-hew/nssp-archival-vintages",
             },
             {
                 "source": "prod-param-estimates",
-                "target": "/pyrenew-hew/nssp_demo/params",
+                "target": "/pyrenew-hew/params",
             },
             {
-                "source": "pyrenew-test-output",
-                "target": "/pyrenew-hew/nssp_demo/private_data",
+                "source": "pyrenew-hew-prod-output",
+                "target": "/pyrenew-hew/output",
+            },
+            {
+                "source": "pyrenew-hew-config",
+                "target": "/pyrenew-hew/config",
             },
         ],
     )
 
     base_call = (
         "/bin/bash -c '"
-        "python nssp_demo/forecast_state.py "
+        "python pipelines/forecast_state.py "
         "--disease {disease} "
         "--state {state} "
         "--n-training-days 75 "
         "--n-warmup 1000 "
         "--n-samples 500 "
-        "--facility-level-nssp-data-dir nssp_demo/nssp-etl/gold "
+        "--facility-level-nssp-data-dir nssp-etl/gold "
         "--state-level-nssp-data-dir "
-        "nssp_demo/nssp-archival-vintages/gold "
-        "--param-data-dir nssp_demo/params "
-        "--output-data-dir nssp_demo/private_data "
+        "nssp-archival-vintages/gold "
+        "--param-data-dir params "
+        "--output-data-dir output "
+        "--priors-path config/prod_priors.py "
         "--report-date {report_date} "
         "--exclude-last-n-days 5 "
-        "--score "
+        "--no-score "
```

Comment:
I don't think this works? You just need to omit the flag.

Reply:
This does work but you could also omit. I like including because it's more explicit.

Reply:
TIL https://docs.python.org/3/library/argparse.html#argparse.BooleanOptionalAction